drivers/gpio/Kconfig.rt1718s

```diff
@@ -0,0 +1,45 @@
+# RT1718S driver configuration options
+
+# Copyright 2022 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config GPIO_RT1718S
```

Comment:
How about using menuconfig here? (so that the options below are nested)

Reply:
Done.